test/fixtures/keys/Makefile

```diff
@@ -648,6 +651,17 @@ I_AM_THE_WALRUS_sha256_signature_signedby_rsa_private_b.sha256: rsa_private_b.pe
 rsa_public_b.pem: rsa_private_b.pem
 	openssl rsa -in rsa_private_b.pem -pubout -out rsa_public_b.pem
 
+rsa_cert_foafssl_b.crt: rsa_private_b.pem
```

Comment:
nit: a comment describing something about what this cert is would be nice, I'm not sure what "foaf" means.

Reply:
Added comments!